### PR TITLE
Nerfs baton throwing

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Removes the knockdown from baton throwing.

## Why It's Good For The Game
Baton throwing is quite strong, this should help it become more of a soul thing rather than a main form of combat.

## Testing
We ball (and we throw batong)

## Changelog
:cl:
tweak: Thrown batons now don't knockdown anymore
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/stunbaton.dm [hunk 1]
```diff
@@ -251,17 +251,13 @@
...
 
 	ADD_TRAIT(L, TRAIT_WAS_BATONNED, user_UID) // so one person cannot hit the same person with two separate batons
-	L.apply_status_effect(STATUS_EFFECT_DELAYED, 2 SECONDS, CALLBACK(L, TYPE_PROC_REF(/mob/living, KnockDown), knockdown_duration), COMSIG_LIVING_CLEAR_STUNS)
 	addtimer(CALLBACK(src, PROC_REF(baton_delay), L, user_UID), 2 SECONDS)
 
 	SEND_SIGNAL(L, COMSIG_LIVING_MINOR_SHOCK, 33)
 
 	L.lastattacker = user.real_name
 	L.lastattackerckey = user.ckey
-	L.visible_message("<span class='danger'>[user] has stunned [L] with [src]!</span>",
```
Comment (Contrabang) [inline code/game/objects/items/weapons/stunbaton.dm:261]: Any specific reason to remove this message?

File: code/game/objects/items/weapons/stunbaton.dm [hunk 2]
```diff
@@ -251,17 +251,13 @@
...
-		"<span class='userdanger'>[L == user ? "You stun yourself" : "[user] has stunned you"] with [src]!</span>")
 	add_attack_logs(user, L, "stunned")
-	playsound(src, 'sound/weapons/egloves.ogg', 50, TRUE, -1)
```
Comment (Contrabang) [inline code/game/objects/items/weapons/stunbaton.dm:264]: Does this sound play twice or something? I feel like this sound is a good feedback that they were hit

Comment (GDNgit) [reply code/game/objects/items/weapons/stunbaton.dm:264]: You'll hear the baton actually hitting them still, this just removes the feedback that makes it pretty much the same as a melee baton strike

Comment (Contrabang) [reply code/game/objects/items/weapons/stunbaton.dm:264]: So unless they noticed the baton flying towards them, why won't have any other feedback? I think some kind of text/sound would be good

Comment (GDNgit) [reply code/game/objects/items/weapons/stunbaton.dm:264]: They hear the baton hitting them, just not the additional sound effect

Comment (Contrabang) [reply code/game/objects/items/weapons/stunbaton.dm:264]: I know, but thats like the same sound as anything hitting them, its not really clear feedback that it did anything different than a normal object hitting them

File: code/game/objects/items/weapons/stunbaton.dm (hunk 1 above)
Comment (DGamerL) [reply code/game/objects/items/weapons/stunbaton.dm:261]: It felt weird seeing these messages in the chat:
```
X has thrown the stunbaton
X has stunned Y with the stunbaton
Y has been hit with the stunbaton
```
Imo the first and last of these are good enough to indicate what happened, but if you disagree I'm also fine with adding it back in

Comment (Contrabang) [reply code/game/objects/items/weapons/stunbaton.dm:261]: I was thinking something like `the stunbaton shocks Y` or something like that, but honestly with the sound-addition back, its fine.

Comment (DGamerL) [reply code/game/objects/items/weapons/stunbaton.dm:261]: Oh good call, I'll add it back in

File: code/game/objects/items/weapons/stunbaton.dm [hunk 3]
```diff
@@ -251,15 +251,13 @@
...
-	L.apply_status_effect(STATUS_EFFECT_DELAYED, 2 SECONDS, CALLBACK(L, TYPE_PROC_REF(/mob/living, KnockDown), knockdown_duration), COMSIG_LIVING_CLEAR_STUNS)
 	addtimer(CALLBACK(src, PROC_REF(baton_delay), L, user_UID), 2 SECONDS)
 
 	SEND_SIGNAL(L, COMSIG_LIVING_MINOR_SHOCK, 33)
 
 	L.lastattacker = user.real_name
 	L.lastattackerckey = user.ckey
-	L.visible_message("<span class='danger'>[user] has stunned [L] with [src]!</span>",
-		"<span class='userdanger'>[L == user ? "You stun yourself" : "[user] has stunned you"] with [src]!</span>")
+	L.visible_message("<span class='danger'>[src] has stunned [L]!</span>")
```
Comment (Contrabang) [inline code/game/objects/items/weapons/stunbaton.dm:260]: Code is fine, but I'm trying to think of a better way of this sounding smooth when all 3 of these lines are read, this is how it is currently. The problem is it switches between past tense and present tense. I think maybe this visiuble message on this line should be in present tense, its just to me "has stunned" sounds a bit awkward.
```
...has thrown...
*has stunned*
...is hit by...
```

**Review by Contrabang — APPROVED** [review]

**Review by Henri215 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**